home/jobs/OMERO-build/config.xml

```diff
@@ -63,7 +63,8 @@ export OMERO_BRANCH=$MERGE_PUSH_BRANCH
 
 virtualenv $WORKSPACE/omero-virtualenv --system-site-packages
 source $WORKSPACE/omero-virtualenv/bin/activate
-pip install -U setuptools
+pip install https://github.com/ome/zeroc-ice-py-manylinux/releases/download/0.1.0/zeroc_ice-3.6.5-cp36-cp36m-manylinux2010_x86_64.whl
```

Comment:
```suggestion
pip install https://github.com/ome/zeroc-ice-py-manylinux/releases/download/0.2.0/zeroc_ice-3.6.5-cp36-cp36m-manylinux2010_x86_64.whl
```
and elsewhere, no idea if it's any better though!

Reply:
The `manylinux2010` wheel bundle a static copy of openssl, so `0.2.0` will have been recompiled against a newer version.

Reply:
@manics: i.e. better to bump or better to omit?

Reply:
If you use https://github.com/ome/zeroc-ice-py-centos7/ the wheel will be dynamically linked to the current openssl library which is safer.